assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { noop, find } from 'lodash';
+import { format, parseISO } from 'date-fns';
+
+import Button from '@common/Button';
+import Table from '@common/Table';
+import PageHeader from '@common/PageHeader';
+import Modal from '@common/Modal';
+import Tooltip from '@common/Tooltip';
+import Banner from '@common/Banners/Banner';
+
+const defaultUsers = [];
+
+function getUserByID(users, userID) {
+  return find(users, { id: userID }) || null;
+}
+
+function Users({
+  onDeleteUser = noop,
+  navigate = noop,
+  users = defaultUsers,
+  loading = false,
+}) {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [deleteUserID, setDeleteUserID] = useState(null);
+
+  const usersTableConfig = {
+    pagination: true,
+    usePadding: false,
+    columns: [
+      {
+        title: 'Username',
+        key: 'username',
+        render: (content, item) => (
+          <Link
+            className="text-jungle-green-500 hover:opacity-75"
+            to={`/users/${item.id}/edit`}
+          >
+            {content}
+          </Link>
+        ),
+      },
+      {
+        title: 'Full Name',
+        key: 'fullname',
+      },
+      {
+        title: 'Email',
+        key: 'email',
+      },
+      {
+        title: 'Status',
+        key: 'enabled',
+        render: (content, item) => (
+          <span>{item.enabled ? 'Enabled' : 'Disabled'}</span>
+        ),
+      },
+      {
+        title: 'Created',
+        key: 'created_at',
+        render: (content, item) => (
+          <span> {format(parseISO(item.created_at), 'MMMM dd, yyyy')} </span>
```

Comment:
We could put `'PPpp'` as format to show the hour and minute.
Anyway, what Jurgen says :stuck_out_tongue_closed_eyes: